website/docs/r/apigatewayv2_integration.html.markdown

```diff
@@ -63,13 +63,37 @@ resource "aws_apigatewayv2_integration" "example" {
 }
 ```
 
+### Private Integration
+
+```hcl
+resource "aws_apigatewayv2_integration" "example" {
+  api_id              = aws_apigatewayv2_api.example.id
+  credentials_arn     = aws_iam_role.example.arn
+  description         = "Example with a load balancer"
+  integration_type    = "HTTP_PROXY"
+  integration_uri     = aws_lb_listener.example.arn
+  
+  connection_type = "VPC_LINK"
+  connection_id   = aws_apigatewayv2_vpc_link.example.id
+
+  request_parameters = {
+    "QueueUrl"    = "$request.header.queueUrl"
+    "MessageBody" = "$request.body.message"
+  }
```

Comment:
I think it's better to remove the `request_parameters` block (particularly with these values) as it is usually applicable to an AWS service integration.

Reply:
Thanks, copy paste issue 👍